bika/lims/upgrade/configure.zcml

```diff
@@ -101,4 +101,12 @@
        handler="bika.lims.upgrade.v01_02_001.upgrade"
        profile="bika.lims:default"/>
 
+ <genericsetup:upgradeDepends
+       description="Allow Link types in the context of an AR"
+       title="Upgrade to SENAITE.CORE 1.2.2"
+       source="1.2.1"
+       destination="1.2.2"
+       import_steps="typeinfo"
+       profile="bika.lims:default"/>
+
```

Comment:
No need to add a new upgrade step. We do this each time we publish a new release. If your changes require some code to be included in an upgrade step, include the code in the upgrade step that matches with the current unreleased version.

Reply:
In this case there is no code change required, just to rerun portal_setup typeinfo.